float64 x = 0.0
float64 y = 0.0
float64 theta = 0.0
---
int64 result